src/js/preshuffle.s

```diff
@@ -893,6 +893,7 @@ StoreObjectExp:
 LevelUp:
   ; double check here that we aren't already max level
   inc PlayerLevel
+CrowdControlRunLevelUp:
```

Comment:
Maybe just call this `UpdatePlayerLevelAfterChange` or something?  It's not necessary specific to crowd-control.